ts/@live-compositor/browser-render/scripts/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "types": ["node"],
```

Comment:
node? this should run in browser

Reply:
This is needed for `buildWasm.ts` script. It uses types from node

Reply:
So maybe rename `buildWasm.ts` -> `buildWasm.js`